### PR TITLE
ISSD-3026 [CP 2.0] Update account roles on Customer Portal

### PR DESCRIPTION
**Pull request by pedroliveirav**
[ISSD-3026](https://liferay.atlassian.net/browse/ISSD-3026)

### CODE REVIEW

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/utils/getProjectRoles.js [hunk 1]
```diff
@@ -36,6 +36,10 @@ export function getRolesFiltered(items, project) {
 						role.name !== ROLE_TYPES.partnerMember.key;
```
Comment (amosfong) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/utils/getProjectRoles.js:36]: Hey Pedro, can you add the new roles to this condition?

Comment (pedroliveirav) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/utils/getProjectRoles.js]: Done.

**Comment by amosfong** [comment]
Pedro,

Can you sort the list alphabetically? Then don't forget to run gradlew formatSource. Thanks

**Comment by amosfong** [comment]
Looks good, thanks!

**Review by amosfong — APPROVED** [review]

**Review by kylembischof — APPROVED** [review]

**Comment by kylembischof** [comment]
ci:forward

**Pull request closed without merging**